### ISSUE TITLE
Tooltip vs HelperText

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystem#182]
Our `HelperText` behaves much like a tooltip with some added sugar (icon + button). 

We should decide on how this component behaves and if it should be just like a tooltip or not in terms of WCAG 2.1. 

related issue #93

**Comment by mimarz** [comment]
Duplicate of #29
